packages/gatsby-core-utils/src/service-lock.ts

```diff
@@ -68,15 +68,19 @@ export const createServiceLock = async (
 
 export const getService = async (
   programPath: string,
-  serviceName: string
-): Promise<string | null> => {
+  serviceName: string,
+  ignoreLockfile: boolean = false
```

Comment:
Doesn't look like the new parameter is used anywhere? 🤔

Reply:
Look at line 81

Reply:
I meant from the outside 😅

Reply:
Oh, that's for me in Desktop! I want to get the data without obtaining a lock